torch/_dynamo/debug_utils.py

```diff
@@ -223,7 +223,9 @@ def _cuda_system_info_comment():
 
     model_str = "# CUDA Info: \n"
     try:
-        cuda_version_out = subprocess.run(["nvcc", "--version"], stdout=subprocess.PIPE)
+        cuda_version_out = subprocess.run(
+            ["nvcc", "--version"], stdout=subprocess.PIPE, check=True
+        )
```

Comment:
```suggestion
        cuda_version_out = subprocess.check_output(
            ["nvcc", "--version"]
        )
```

Reply:
Not sure why you've resolved this one, as imo it's cleaner to replace it with `check_output` , as [doc](https://docs.python.org/3/library/subprocess.html#subprocess.check_output) literally says(you'll have to remove stdout from the next line, I just can do it from web-ui):
> This is equivalent to:
> ```
> run(..., check=True, stdout=PIPE).stdout
> ```

Reply:
@malfet I have to change the semantic because it reads the stdout in the next line so this change isn't quite valid.